addons/escoria-core/game/core-scripts/esc/commands/stop_snd.gd

```diff
@@ -54,7 +54,9 @@ func validate(arguments: Array):
 
 # Run the command
 func run(command_params: Array) -> int:
-	previous_snd_state = escoria.object_manager.get_object(command_params[0]).node.state
+	var sound_player = escoria.object_manager.get_object(command_params[0]).node
```

Comment:
Is it at all possible for that `node` to be null/invalid? If it can be, then the line after this may cause an error.

Reply:
Sound player should never be null.